thrift/cadence.thrift

```diff
@@ -137,6 +138,7 @@ service WorkflowService {
       5: shared.EntityNotExistsError entityNotExistError,
       6: shared.DomainNotActiveError domainNotActiveError,
       7: shared.ClientVersionNotSupportedError clientVersionNotSupportedError,
+      8: shared.WorkflowExecutionAlreadyCompletedError workflowExecutionAlreadyCompletedError,
```

Comment:
Does this API really need this error?

Reply:
Seems like PollForDecisionTask should handle the error instead of passing it to the caller